session.rst

```diff
@@ -26,7 +26,7 @@ You need to install the HttpFoundation component to handle sessions:
 
 .. code-block:: terminal
 
-    $ composer require symfony/http-foundation
+    composer require symfony/http-foundation
```

Comment:
Please revert this, maybe @javiereguiluz can modify the copy button for this special case?

I also added a rule to DOCtor-RST the we ensure to keep this `$`-prompt for composer commands.

https://github.com/OskarStark/doctor-rst/pull/1307

Reply:
But that's already the case. You can't select this prompt. See:

![text-selection-prompt](https://user-images.githubusercontent.com/73419/217467867-4f403611-52cf-4f5b-adb3-bebf726594a8.gif)

Reply:
Ah I thought we have a copy button next to it on the right side

Reply:
On the pages where we show a "Copy" button (e.g. https://symfony.com/download) we also remove the prompt from the copied contents.